src/jquery.mockjax.js

```diff
@@ -269,6 +269,7 @@
 				url: mockHandler.proxy,
 				type: mockHandler.proxyType,
 				data: mockHandler.data,
+				async: false,
```

Comment:
I'm not sure why you force the proxy ajax call to by synchronous here. If the original request was synchronous, then we could do that I suppose, but we wouldn't want _all_ proxy calls to be synchronous. Consider changing this to match he original settings.